src/init.jl

```diff
@@ -0,0 +1 @@
+
```

Comment:
Did you mean to put some content in this file?

Reply:
Nope it should be deleted.